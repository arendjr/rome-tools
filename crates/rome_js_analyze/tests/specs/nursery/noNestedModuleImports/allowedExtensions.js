/* should not generate diagnostics */

import "./resources/allowed_icon.png";
