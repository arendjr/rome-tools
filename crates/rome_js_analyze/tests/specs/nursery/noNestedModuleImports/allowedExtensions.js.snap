---
source: crates/rome_js_analyze/tests/spec_tests.rs
expression: allowedExtensions.js
---
# Input
```js
/* should not generate diagnostics */

import "./resources/allowed_icon.png";

```


